internal/featureflags/featureflags.go

```diff
@@ -0,0 +1,27 @@
+package featureflags
+
+var f = &flags{
+	v1beta1Enabled:         false,
+	logpipelineOTLPEnabled: false,
+}
+
+type flags struct {
+	v1beta1Enabled         bool
+	logpipelineOTLPEnabled bool
+}
+
+func SetV1beta1Enabled(enabled bool) {
+	f.v1beta1Enabled = enabled
+}
+
+func IsV1beta1Enabled() bool {
+	return f.v1beta1Enabled
+}
```

Comment:
nit: I think in other places we use `V1Beta1`, could you please check?

Reply:
we use v1Beta1 ... updated accordingly